### ISSUE TITLE
v.1.8.7 - border of other chosen list is appearing if we make any chosen dropdown disabled. I am using IE 11 browser.

### ISSUE THREAD

**Issue by emailajay21** [harvesthq/chosen#3041]
Please check this link 
https://www.w3schools.com/code/tryit.asp?filename=FZEDONKVKO5Q

**Comment by emailajay21** [comment]
This issue only occurs when we have multiple chosen dropdown on page and we try to disable last dropdown.

**Comment by ffaubert** [comment]
Any suggestions on how to fix this?

**Comment by emailajay21** [comment]
Yes I got some work around. Just add another dropdown and set it's visibility to hidden. It will work.

**Comment by echart** [comment]
Hi guys, for my use case this solved the issue:
`.chosen-container:not(.chosen-with-drop) .chosen-drop {
  display: none;
}
.chosen-container.chosen-with-drop .chosen-drop {
  display: block;
}`
